### PR TITLE
VGP TrimNs tool

### PR DESCRIPTION
**Pull request by astrovsky01**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/TrimNs/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: TrimNs
```
Comment (bgruening) [inline tools/TrimNs/.shed.yml:1]: small letters

File: tools/TrimNs/TrimNs.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<tool id="TrimN" name="TrimN" version="0.1.0" profile="20.01" python_template_version="3.4">
```
Comment (bgruening) [inline tools/TrimNs/TrimNs.xml:1]: tool-id please small letters

File: tools/TrimNs/TrimNs.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <requirements>
+        <requirement type="package" version="3.7">python</requirement>
```
Comment (bgruening) [inline tools/TrimNs/TrimNs.xml:3]: I think you can skip here python, it comes with biopython

File: tools/TrimNs/clip_regions_DNAnexus.py [hunk 4]
```diff
@@ -0,0 +1,245 @@
+import os
```
Comment (bgruening) [inline tools/TrimNs/clip_regions_DNAnexus.py:1]: Do you know which license this code has?

Comment (astrovsky01) [reply tools/TrimNs/clip_regions_DNAnexus.py:1]: It's a non-standard license file: https://github.com/VGP/vgp-assembly/blob/master/LICENSE

Comment (bgruening) [reply tools/TrimNs/clip_regions_DNAnexus.py:1]: looks like a BSD-3 license, isn't it? https://opensource.org/licenses/BSD-3-Clause

If you agree, can you please add a license="BSD-3-Clause" tag to the tool? Thanks.

**Comment by bgruening** [comment]
The tool linting of the python code is failing. I guess we have two options. Either fix the upstream code or create a conda package with just those 3 scripts?

File: tools/TrimNs/clip_regions_DNAnexus.py (hunk 4 above)
Comment (astrovsky01) [reply tools/TrimNs/clip_regions_DNAnexus.py:1]: Oops, my bad. Yeah, will do

**Comment by bernt-matthias** [comment]
+1 for creating a conda package. According to the readme parts of this repo already are in conda.

**Comment by bgruening** [comment]
@bernt-matthias it seems that no tests have been run, at least according to the artifacts summary? Is that a planemo bug o GHA bug?

File: tools/TrimNs/TrimNs.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<tool id="trimns" name="TrimN" version="0.1.0" profile="21.01" license="BSD-3-Clause">
```
Comment (bernt-matthias) [inline tools/TrimNs/TrimNs.xml:1]: > @bernt-matthias it seems that no tests have been run, at least according to the artifacts summary? Is that a planemo bug o GHA bug?

```
Exception: The tool trimns targets version 21.01 of Galaxy, you should upgrade Galaxy to ensure proper functioning of this tool.
```

The CI workflow runs 20.09 and therefor the tool is not tested. So we need a profile <= 20.09

Comment (bgruening) [reply tools/TrimNs/TrimNs.xml:1]: But this should fail the CI left?

Comment (bernt-matthias) [reply tools/TrimNs/TrimNs.xml]: It should. I will open an issue for planemo.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Lets ship it :)

VGP here we come.

**Pull request merged**